rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -601,26 +702,31 @@ error_code cellGemGetImageState(u32 gem_num, vm::ptr<CellGemImageState> image_st
 		return CELL_GEM_ERROR_UNINITIALIZED;
 	}
 
-	if (!check_gem_num(gem_num) || !image_state)
+	if (!check_gem_num(gem_num) || !gem_image_state)
 	{
 		return CELL_GEM_ERROR_INVALID_PARAMETER;
 	}
 
+	auto shared_data = g_fxo->get<gem_camera_shared>();
+
 	if (g_cfg.io.move == move_handler::fake)
 	{
-		auto shared_data = g_fxo->get<gem_camera_shared>();
+		gem_image_state->u = 0;
+		gem_image_state->v = 0;
+		gem_image_state->projectionx = 1;
+		gem_image_state->projectiony = 1;
+	}
+	else if (g_cfg.io.move == move_handler::mouse)
+		mouse_pos_to_gem_image_state(gem_num, gem_image_state);
 
-		image_state->frame_timestamp = shared_data->frame_timestamp.load();
-		image_state->timestamp = image_state->frame_timestamp + 10;   // arbitrarily define 10 usecs of frame processing
-		image_state->visible = true;
-		image_state->u = 0;
-		image_state->v = 0;
-		image_state->r = 20;
-		image_state->r_valid = true;
-		image_state->distance = 2 * 1000;   // 2 meters away from camera
-		// TODO
-		image_state->projectionx = 1;
-		image_state->projectiony = 1;
+	if (!(g_cfg.io.move == move_handler::null))
```

Comment:
Remove this check and always execute the code below.
Move the nulll handler check above the other move handler checks and return CELL_OK if move_handler::null

Reply:
in master is already like that, run this only with fake, i have just set same